workflows/cellranger-quant/run-cellranger.nf

```diff
@@ -12,6 +12,9 @@ params.run_metafile = 's3://ccdl-scpca-data/sample_info/scpca-library-metadata.t
 params.run_ids = "SCPCR000001,SCPCR000002"
 params.outdir = 's3://nextflow-ccdl-results/scpca/cellranger-quant'
 
+// technology options
+technology = ["10Xv2", "10Xv3", "10Xv3.1"]
```

Comment:
Minor thing that I try to do, but you might not care about, is that if there is a list/vector I try to give it a plural name or some kind of suffix to indicate the data structure.
```suggestion
tech_list = ["10Xv2", "10Xv3", "10Xv3.1"]
```